sdk/python/feast/infra/feature_servers/multicloud/requirements.txt

```diff
@@ -0,0 +1 @@
+feast[aws,gcp,snowflake,redis,go,mysql,postgres,opentelemetry,grpcio,k8s,duckdb,milvus] == ${VERSION}
```

Comment:
Just a thought : How about having OPERATOR_REQUIRED/MULTICLOUD_REQUIRED in 
https://github.com/feast-dev/feast/blob/master/setup.py#L161 and then make command to install those dependencies in Dockerfile instead ?

Reply:
ah, please ignore, seems it is needed for konflux